src/TextField/TextField.jsx

```diff
@@ -24,6 +24,9 @@ function isValid(value) {
 const TextField = React.createClass({
 
   propTypes: {
+    /**
+     * @ignore
```

Comment:
yes @mbrookes I'm also pretty confused with this one so I thought let's remove it and get review from you guys.